File: Desafio3 - GFT/src/model/Veiculo.java
package model;

public class Veiculo {

	private String Marca;
	private String Modelo;
	private boolean isLigado = true;
	private int litrosCombustivel;
	private int Velocidade;
	private int passageiros;
	
	public String getMarca() {
		return Marca;
	}
	public void setMarca(String marca) {
		Marca = marca;
	}
	public String getModelo() {
		return Modelo;
	}
	public void setModelo(String modelo) {
		Modelo = modelo;
	}
	public boolean getLigado() {
		return isLigado;
	}
	public void setLigado(boolean isLigado) {
		this.isLigado = isLigado;
	}
	public int getLitrosCombustivel() {
		return litrosCombustivel;
	}
	public void setLitrosCombustivel(int litrosCombustivel) {
		this.litrosCombustivel = litrosCombustivel;
	}
	public int getVelocidade() {
		return Velocidade;
	}
	public void setVelocidade(int velocidade) {
		Velocidade = velocidade;
	}
	public int getPassageiros() {
		return passageiros;
	}
	public void setPassageiros(int passageiros) {
		this.passageiros = passageiros;
	}
	
	public void acelerar() {
		System.out.println("Acelerando");
	}
	
	public void abastecer(int qtdLitros) {
		System.out.println("Veículo abastecido");
	}
		
	public void desligar() {
		if (getLigado() == true) {
			setLigado(false);
			System.out.println("Veículo desligado");
		} else {
			System.out.println("Veículo já se encontra desligado");
		}
	}
	
}

File: C-Sharp/Desafio03/Desafio03/com.gft.model/Veiculo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Desafio03.com.gft.model
{
    class Veiculo
    {
		private String _marca;
		private String _modelo;
		private bool _isLigado = true;
		private int _litrosCombustivel;
		private int _velocidade;
		private int _passageiros;

		public string Marca { get => _marca; set => _marca = value; }
		public string Modelo { get => _modelo; set => _modelo = value; }
		public bool IsLigado { get => _isLigado; set => _isLigado = value; }
		public int LitrosCombustivel { get => _litrosCombustivel; set => _litrosCombustivel = value; }
		public int Velocidade { get => _velocidade; set => _velocidade = value; }
		public int Passageiros { get => _passageiros; set => _passageiros = value; }

		public virtual void acelerar()
		{
			Console.WriteLine("Acelerando");
		}

		public virtual void abastecer(int qtdLitros)
		{
			Console.WriteLine("Veículo abastecido");
		}

		public virtual void desligar()
		{
			if (IsLigado == true)
			{
				IsLigado = false;
				Console.WriteLine("Veículo desligado");
			}
			else
			{
				Console.WriteLine("Veículo já se encontra desligado");
			}
		}
	}
}
